packages/components/src/feature_alert/index.module.scss

```diff
@@ -0,0 +1,100 @@
+@import 'nebenan-ui-kit';
+
+$feature-alert-arrow-size: 16px;
+
+@keyframes feature-alert-appear {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+.container {
+  width: 300px;
+  display: none;
```

Comment:
.isActive isn't needed. We only render the tooltip if it is active, so it's always shown as 'block'